scripts/variantstore/wdl/GvsImportGenomes.wdl

```diff
@@ -204,6 +206,7 @@ task LoadData {
 
     File? gatk_override
     Int? load_data_preemptible_override
+    Int? load_data_maxretries_override
```

Comment:
Do we have rule for why this would be optional?
You could write it as `Int load_data_maxretries_override = 3`
and then you wouldn't need the select_first in the runtime block

Reply:
I don't have a strong opinion on this, I was following the existing pattern used for `load_data_preemptible_override`. Happy to discuss best practices!

Reply:
yes and I think we were using GATK patterns, but it does seem superfluous to me

Reply:
Do we think there would ever be a reason to retry with more mem? If yes I suppose we should add it to the Terra Quickstart

Reply:
My very limited experience is exclusively with the Quickstart and I don't think that would have benefited from more memory here, but if you're seeing something different with more "real" data sets we could certainly add that.